packages/slice-machine/src/features/customTypes/customTypesBuilder/PageSnippetDialog/PageSnippetDialog.tsx

```diff
@@ -27,11 +31,26 @@ const PageSnippetContent: FC<PageSnippetContentProps> = ({ model }) => {
     return null;
   }
 
+  const trackOpenSnippet = () => {
+    void telemetry.track({
+      event: "page-type:open-snippet",
+      ...(config !== undefined
+        ? {
+            framework: config.adapter,
```

Comment:
This is not good I think, I saw adapter may not be a string, I create a new manager function to just get the adapter name, maybe we could merge my PR first and quickly fetch directly the string adapter name without have to all the config here